### ISSUE TITLE
fix: Narrator Focus is not in sync with cursor when using text field.

### ISSUE THREAD

**Issue by shachak** [microsoft/fast#5502]
# 🐛 Bug Report

## 💻 Repro or Code Sample

Repro-ed in component explorer.

## 🤔 Expected Behavior

narrator focus should be on the cursor. 
## 😯 Current Behavior
While in edit mode in a textField the narrator focus shifts to the next element. 

## 🌍 Your Environment

win11 Edge Browser

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by shachak** [comment]
@chrisdholt this is based on offline discussion we had.

**Comment by chrisdholt** [comment]
@shachak we'll need to likely repro this outside of the component explorer right now. This bug is interesting because it's just an input with type of text for `text-field`. @scomea are you able to see if this repro's in narrator?

**Comment by shachak** [comment]
@chrisdholt you can repro it on any of the msedge text-fields, like edit favorites. But I notice something new, that this is now also repro-ing for a general <input> with type=text, which was not the case before. Also, this is for narrator.

**Comment by scomea** [comment]
This happens with all inputs as far as I can tell, even on the main Google and Bing search boxes.  Not a fast issue.

**Comment by chrisdholt** [comment]
I also find that this seems to repro with native elements - that's not too surprising as the element being focused is an input :). Closing this as external - @shachak I followed up the other evening in our offline thread. Thanks!
